sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -101,4 +101,67 @@ class QueryCompilationErrorsSuite extends QueryTest with SharedSparkSession {
     assert(e.message ===
       "The argument_index of string format cannot contain position 0$.")
   }
+
+  test("CANNOT_USE_MIXTURE") {
```

Comment:
Please, look at other test titles. They contain an error class and some message which explain what the particular test checks. Could you add some message here (and below tests) too, please.